src/components/tabs/tabs.pw.tsx

```diff
@@ -0,0 +1,822 @@
+        await mount(<TabsComponent variant={variant} />);
+
+        await expect(tabById(page, 1)).toHaveCSS(
+          "background-color",
+          backgroundColor
+        );
+      });
+    }
+  );
+
+  [
+    CHARACTERS.STANDARD,
+    CHARACTERS.DIACRITICS,
+    CHARACTERS.SPECIALCHARACTERS,
+  ].forEach(([text]) => {
+    test(`should render Tabs with title prop set to ${text}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<TabsComponent title={text} />);
+
+      await expect(tabById(page, 1)).toHaveText(text);
+    });
+  });
+
+  test("should render Tabs with correct tabId", async ({ mount, page }) => {
+    await mount(<TabsComponent tabId={CHARACTERS.STANDARD} />);
+
+    await expect(getDataElementByValue(page, "select-tab").nth(0)).toHaveId(
+      `${CHARACTERS.STANDARD}-tab`
+    );
+  });
+
+  test("should render Tabs with customLayout prop", async ({ mount, page }) => {
+    await mount(<WithCustomLayout />);
+
+    await expect(tabById(page, 1).locator(ICON).nth(0)).toBeVisible();
+    await expect(tabById(page, 1).locator(ICON).nth(1)).toBeVisible();
+    await expect(page.getByText("Tab 1", { exact: true })).toBeVisible();
+  });
+
+  test("should render Tabs with siblings prop", async ({ mount, page }) => {
+    await mount(<WithAdditionalTitleSiblings />);
+
+    const pill = tabById(page, 1).locator(PILL_PREVIEW);
+
+    await expect(pill).toBeVisible();
+    await expect(pill).toHaveText("12");
+  });
+
+  ([
+    ["before", 1, "left"],
+    ["after", 3, "right"],
+  ] as [TabProps["titlePosition"], number, string][]).forEach(
+    ([pos, id, style]) => {
+      test(`should render Tabs with titlePosition prop set to ${pos}`, async ({
+        mount,
+        page,
+      }) => {
+        await mount(<WithAdditionalTitleSiblings />);
+
+        const layoutWrapper = tabById(page, id)
+          .locator(PILL_PREVIEW)
+          .locator("..");
+
+        await expect(layoutWrapper).toHaveCSS(style, "8px");
+      });
+    }
+  );
+
+  ([
+    [1, validationType[0]],
+    [2, validationType[1]],
+    [3, validationType[2]],
+  ] as [number, string][]).forEach(([id, validation]) => {
+    test(`should render Tab ${id} with ${validation}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<TabsComponentValidations />);
+
+      await expect(tabById(page, id).locator(ICON)).toHaveAttribute(
+        "type",
+        validation
+      );
+    });
+  });
+
+  ([
+    [1, validationType[0]],
+    [2, validationType[1]],
+    [3, validationType[2]],
+  ] as [number, string][]).forEach(([id, validation]) => {
+    test(`should display ${validation} message when hovering over Tab ${id}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<TabsComponentValidations />);
+
+      await tabById(page, id).hover();
+
+      await expect(tooltipPreview(page)).toHaveText(validation);
+    });
+  });
+
+  [validationType[0], validationType[1], validationType[2]].forEach(
```

Comment:
**suggestion(non-blocking):** you shouldn't need to specify the index of the array. I think this was how it was done in Cypress but we can just do this now I think.

```suggestion
  validationType.forEach(
```
or if you implement my change to the const array declaration. 

```suggestion
  validationTypes.forEach(
```